doc/source/tutorial/weave.rst

```diff
@@ -2520,10 +2520,10 @@ cryptic error report due to the fact that ``stdio.h`` also defines the name.
 ``weave`` should probably try and handle this in some way. Other things...
 
 .. _PyInline: http://pyinline.sourceforge.net/
-.. _SciPy: http://www.scipy.org
+.. _SciPy: https://www.scipy.org
 .. _mingw32: http://www.mingw.org%3Ewww.mingw.org
 .. _NumPy: http://numeric.scipy.org/
```

Comment:
Sorry, I was looking at the wrong line. It looks like http://numeric.scipy.org/ and https://numeric.scipy.org/ serve different content though, so perhaps securing that one is not ideal?